src/simulator/system_parameters.py

```diff
@@ -40,20 +40,21 @@ class SystemParameters(NamedTuple):
     """Parameters defining a simulation run.
 
     Attributes:
-        campaign_spend:  A list of spend values, one per campaign,
-            representing the total actual spend for each campaign.
+        campaign_spend_fraction:  A list of values, each between 0 and 1,
+            one per campaign, representing the amount spent on the campaign
```

Comment:
I would call out that len(campaign_spend_fractions) = number of publishers

Reply:
Done.